specification/search/resource-manager/readme.md

```diff
@@ -26,7 +26,7 @@ These are the global settings for the SearchManagementClient API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2023-11
```

Comment:
why you change the apiversion to a older preview one?

Reply:
The new stable version should still be there.  My understanding is that all stable versions show up in the docs automatically, and if some of those versions aren't needed, they're suppressed during the doc autogen process.  This fix is an attempt to restore preview APIs docs, which we used to have, but disappeared when the platform team adopted the new filtering mechanism.   I was told that if I list the preview API package in the readme. I'll get the preview API docs back.  That's what I'm trying to accomplish here.